index.html

```diff
@@ -0,0 +1,89 @@
+<!doctype html>
+<!--
+Copyright 2018 The Go Authors. All rights reserved.
+Use of this source code is governed by a BSD-style
+license that can be found in the LICENSE file.
+-->
+<html>
+
+<head>
+	<meta charset="utf-8">
+	<title>Go wasm</title>
+</head>
+
+<body>
+	<!--
+	Add the following polyfill for Microsoft Edge 17/18 support:
+	<script src="https://cdn.jsdelivr.net/npm/text-encoding@0.7.0/lib/encoding.min.js"></script>
+	(see https://caniuse.com/#feat=textencoder)
+	-->
+	<script src="wasm_exec.js"></script>
+	<script>
+		if (!WebAssembly.instantiateStreaming) { // polyfill
+			WebAssembly.instantiateStreaming = async (resp, importObject) => {
+				const source = await (await resp).arrayBuffer();
+				return await WebAssembly.instantiate(source, importObject);
+			};
+		}
+
+		let exitCode = 0;
+		function goExit(code) {
+			exitCode = code;
+		}
+		function enosys() {
+			const err = new Error("not implemented");
+			err.code = "ENOSYS";
+			return err;
+		}
+		let coverageProfileContents;
+
+		(async() => {
+			const go = new Go();
+			const defaultWriteSync = window.fs.writeSync;
```

Comment:
Let's change all references to `window` to `globalThis`.

Reply:
Fixed in [722a2b6](https://github.com/agnivade/wasmbrowsertest/pull/41/commits/722a2b6cc6ad62e87cc275a0e906b7265830aaf7)